README.md

```diff
@@ -87,4 +89,12 @@ To debug tests from CLI:
 To update fixtures:
 - Make required changes in the installed fixture site
 - Run `ahoy drush cex -y` for Drupal 8/9 or `ahoy drush fua -y` for Drupal 7
-- Run `ahoy update-fixtures` for Drupal 8, or `DRUPAL_VERSION=9 ahoy update-fixtures` for Drupal 9, or `DRUPAL_VERSION=7 ahoy update-fixtures` for Drupal 7 to export configuration changes from build directory to the fixtures directory. 
+- Run `ahoy update-fixtures` for Drupal 8, or `DRUPAL_VERSION=9 ahoy update-fixtures` for Drupal 9, or `DRUPAL_VERSION=7 ahoy update-fixtures` for Drupal 7 to export configuration changes from build directory to the fixtures directory.
+
+#### Writing / Updating tests
+
+When developing new test features that to commit new tests, test features must be updated in the
```

Comment:
this has been fixed - the tests are now mounted into container from `./test` on host, so this is no longer required. please remove